buildrunner/validation/config.py

```diff
@@ -188,5 +157,5 @@ def validate_config(**kwargs) -> Errors:
     try:
         Config(**kwargs)
     except ValidationError as exc:
-        errors = _add_validation_errors(exc)
+        errors = add_validation_errors(exc)
```

Comment:
Is this really *adding* validation errors or just getting validation errors?

Reply:
Yes, you are correct. It really just gets the errors. I'll make the change.